CONTRIBUTING.md

```diff
@@ -79,6 +79,9 @@ Prior to running 'bin/setup', it may be necessary to launch the postgres server
 
 To get started, run `bin/setup`
 
+if bin/setup fails check your database.yml file and make sure that you have postgres credentials (i.e.,username:<uname>, password:<pw>)
```

Comment:
Maybe we could provide more details about what sort of error the person would see? There are a lot of things that happen in `bin/setup`, so would want to narrow it down a bit. Minor: Also, mind capitalizing the first letter?